server.py

```diff
@@ -0,0 +1,122 @@
+import re
+import socket
+import threading
+from datetime import datetime
+from logger import logger
+from users_db import Database
+
+
+ADDRESS = "127.0.0.1"
+PORT = 5050
+
+
+class SocketServer:
+    def __init__(self, address, port, db_path):
+        self.address = address
+        self.port = port
+        self.db_path = db_path
+        self.server = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        self.clients_connected = []
+        self.usernames_connected = []
+
+    FORMAT = "utf-8"
+    DATE_FORM = "%d/%m/%Y %H:%M:%S"
+    WELCOME_MSG = "Welcome to the server. Please, type '/help' to get a list of available commands"
+    HELP_MSG = """List of available commands:
```

Comment:
class variables should be above the constructor

Reply:
Fixed